doc/releases/changelog-dev.md

```diff
@@ -793,6 +793,9 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* Revert [(#6933)](https://github.com/PennyLaneAI/pennylane/pull/6933) to remove non-negligible performance impact due to wire flattening.
```

Comment:
Would it be better to remove both this line and the one for #6933 from the changelog? Just to cut down on confusion since this feature didn't appear in v0.40

Reply:
It's not a bad idea, and I could be convinced either way. Usually I think it is better to avoid rewriting history, and to keep everything preserved.
We can leave that decision to product at release time though :)